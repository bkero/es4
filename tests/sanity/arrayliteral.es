intrinsic::assert (([,10,,"hello",,])[3] == "hello")
print ("PASSED!")
