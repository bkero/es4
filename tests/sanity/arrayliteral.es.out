/Applications/smlnj/bin/sml @SMLload=es4.image.x86-darwin -e  tests/sanity/arrayliteral.es
parsing ... 
defining ... 
verifying ... 
evaluating ... 
PASSED!
